### PR TITLE
DOCS-3910 Advanced Options for Browser Test Steps Edits 

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Updates the Advanced Options for browser test details.

### Motivation

DOCS-3910

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/browser_tests/advanced_options.md [hunk 1]
```diff
@@ -6,20 +6,27 @@ aliases:
...
+  tag: "Blog"
+  text: "User experience monitoring with Synthetic browser tests"
 - link: "/synthetics/browser_tests/actions/"
   tag: "Documentation"
   text: "Learn more about Browser Test Steps"
 ---
 
 ## Overview
 
+This page describes how Synthetic browser tests are self maintained and the various advanced options available in the recording workflow.
```
Comment (HugoPct) [inline content/en/synthetics/browser_tests/advanced_options.md:19]: is it a good practice to have the 1st sentence of a page saying "This page describes ..."

**Review by HugoPct — CHANGES_REQUESTED** [review]

File: content/en/synthetics/browser_tests/advanced_options.md (hunk 1 above)
Comment (alai97) [reply content/en/synthetics/browser_tests/advanced_options.md:19]: Yep, it's a common practice across our doc set to basically introduce the topics in this doc page.

Comment (alai97) [reply content/en/synthetics/browser_tests/advanced_options.md:19]: How about something like this?

```suggestion
This page describes how Synthetic browser tests locate elements with the Datadog algorithm and the Datadog user locator system. 
```

File: content/en/synthetics/browser_tests/advanced_options.md [hunk 2]
```diff
@@ -6,20 +6,27 @@ aliases:
...
 
 To prevent flaky tests, Datadog uses an algorithm that leverages a set of locators to target elements in browser tests. A small change in the UI may modify an element (for example, moving it to another location). The browser test automatically locates the element again based on points of reference that were not affected by the change. 
 
 When the test runs successfully, the browser test recomputes (or "self heals") any broken locators with updated values, ensuring your tests do not break from simple UI updates and that your tests are automatically adapting to your application's UI. 
 
-To ensure that your browser test does not validate an unexpected change, use assertions in your test creation. Assertions allow you to define what is and what is not expected behavior associated with the test step journey. 
+To ensure that your browser test does not validate an unexpected change, use [assertions][5] in your test creation. Assertions allow you to define what is and what is not expected behavior associated with the test step journey. 
 
 ## User specified locator
```
Comment (alai97) [inline content/en/synthetics/browser_tests/advanced_options.md:31]:
```suggestion
### User specified locator
```

File: content/en/synthetics/browser_tests/_index.md [hunk 3]
```diff
@@ -96,8 +102,10 @@ Define the configuration of your browser test.
 6. Select **managed and private locations**: Select locations around the world that are managed by Datadog or create [private locations][1] to run your browser test from custom locations or inside private networks. 
 
    {{% managed-locations %}} 
+
+   For more information about using the `datadog-ci` NPM package to trigger tests on a private application, see the [Continuous Testing Tunnel][15].
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/synthetics/browser_tests/_index.md:106]:
```suggestion
   You can also use the [Continuous Testing Tunnel][15] to easily trigger tests on your local development setup or in your CI/CD pipeline to test internal environments. 
```

File: content/en/synthetics/browser_tests/advanced_options.md [hunk 4]
```diff
@@ -6,22 +6,32 @@ aliases:
...
 ---
 
 ## Overview
 
+This page describes how Synthetic browser tests are self maintained and the various advanced options available in the recording workflow. 
+
+
+## Locate an element
+
+## Datadog algorithm
```
Comment (HugoPct) [inline content/en/synthetics/browser_tests/advanced_options.md:24]:
```suggestion
### Datadog algorithm
```

File: content/en/synthetics/browser_tests/advanced_options.md (hunk 1 above)
Comment (HugoPct) [reply content/en/synthetics/browser_tests/advanced_options.md:19]: yup and if you think that doesn't bring any value we should probably drop it

File: content/en/synthetics/browser_tests/_index.md [hunk 5]
```diff
@@ -96,6 +102,8 @@ Define the configuration of your browser test.
 6. Select **managed and private locations**: Select locations around the world that are managed by Datadog or create [private locations][1] to run your browser test from custom locations or inside private networks. 
 
    {{% managed-locations %}} 
+
+   You can also use the [Continuous Testing Tunnel][15] to easily trigger tests on your local development setup or in your CI/CD pipeline to test internal environments.
```
Comment (cswatt) [inline content/en/synthetics/browser_tests/_index.md:106]:
```suggestion
   You can also use the [Continuous Testing Tunnel][15] to trigger tests on your local development setup or in your CI/CD pipeline to test internal environments. 
```

**Review by cswatt — CHANGES_REQUESTED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**